Game/Source/GDKTestGyms/Private/GasCube.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "GasCube.h"
+
+AGasCube::AGasCube()
```

Comment:
This cube isn't a cube at all! Where's the static mesh?

Reply:
Ehh good point. The static mesh currently lives in the blueprint asset, which I think is the right place. Will rename this to just "AGasActor", as it is just that. (Also the doc comment for the class is wrong, will also fix)